pyproject.toml

```diff
@@ -41,7 +41,7 @@ importlib_resources = ">=5.4.0" #Needed to copy resource files, can be removed a
 "stopwatch.py" = ">=1.0.0"
 fabric = "^3.0.1"
 portalocker = "^2.7.0"
-exasol-error-reporting = ">=0.4.0"
+exasol-error-reporting = "^1.0.0"
```

Comment:
this will force projects to go to this, do we still have projects that is pinned to an older version, but in general, but at least these should still be able to use the older itde

Reply:
Oh, you are right